<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow</class>
 <widget class="QMainWindow" name="userWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>userWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="buyerTab">
       <attribute name="title">
        <string>我是买家</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLineEdit" name="buyerSearchLineEdit"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="buyerSearchPushButton">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="buyerViewAllPushButton">
            <property name="text">
             <string>显示在架商品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buyerBuyPushButton">
            <property name="text">
             <string>购买商品</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTableWidget" name="buyerCommodityTableWidget">
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>价格</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>描述</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>状态</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sellerTab">
       <attribute name="title">
        <string>我是卖家</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableWidget" name="sellerCommodityTableWidget">
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>价格</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>描述</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>状态</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="sellerViewAllPushButton">
            <property name="text">
             <string>显示我的商品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sellerNewCommodityPushButton">
            <property name="text">
             <string>发布商品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sellerModifyPushButton">
            <property name="text">
             <string>修改商品信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sellerBanPushButton">
            <property name="text">
             <string>下架商品</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="orderTab">
       <attribute name="title">
        <string>历史订单</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="orderTableWidget">
          <column>
           <property name="text">
            <string>订单ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>商品ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>价格</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>交易时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>卖家ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>买家ID</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="showOrderPushButton">
          <property name="text">
           <string>查看订单</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>buyerCommodityTableWidget</tabstop>
  <tabstop>buyerViewAllPushButton</tabstop>
  <tabstop>buyerBuyPushButton</tabstop>
  <tabstop>buyerSearchLineEdit</tabstop>
  <tabstop>buyerSearchPushButton</tabstop>
  <tabstop>sellerCommodityTableWidget</tabstop>
  <tabstop>sellerViewAllPushButton</tabstop>
  <tabstop>sellerNewCommodityPushButton</tabstop>
  <tabstop>sellerModifyPushButton</tabstop>
  <tabstop>sellerBanPushButton</tabstop>
  <tabstop>orderTableWidget</tabstop>
  <tabstop>showOrderPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buyerSearchPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onBuyerSearchPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buyerBuyPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onBuyerBuyPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buyerViewAllPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onBuyerViewAllPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showOrderPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onshowOrderPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellerViewAllPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onSellerViewAllPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellerNewCommodityPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onSellerNewCommodityPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellerModifyPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onSellerModifyPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sellerBanPushButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindow</receiver>
   <slot>onSellerBanPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBuyerSearchPushButtonClicked()</slot>
  <slot>onBuyerBuyPushButtonClicked()</slot>
  <slot>onBuyerViewAllPushButtonClicked()</slot>
  <slot>onshowOrderPushButtonClicked()</slot>
  <slot>onSellerViewAllPushButtonClicked()</slot>
  <slot>onSellerNewCommodityPushButtonClicked()</slot>
  <slot>onSellerModifyPushButtonClicked()</slot>
  <slot>onSellerBanPushButtonClicked()</slot>
 </slots>
</ui>
